docs/feature_macros.md

```diff
@@ -230,7 +230,7 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
 void matrix_scan_user(void) {     # The very important timer. 
   if (is_alt_tab_active) {
     if (timer_elapsed(alt_tab_timer) > 1000) {
-      unregister_code16(LALT(KC_TAB));
+      unregister_code(KC_LALT);
```

Comment:
Don't forget to unregister TAB here, as well.

Reply:
The Tab key isn't being held, only tapped, so there's no need.

Reply:
Yup, line 223 already has `unregister_code(KC_TAB)` whenever the magic key is released.